CITATION.cff

```diff
@@ -0,0 +1,15 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "Bocchino"
+  given-names: "Robert"
+- family-names: "Canham"
+  given-names: "Timothy"
+- family-names: "Watney"
+  given-names: "Garth"
+- family-names: "Reder"
+  given-names: "Leonard"
+- family-names: "Levison"
+  given-names: "Jeffrey"
+title: "F': A Flight-Proven, Multi-Platform, Open-Source Flight Software Framework"
+url: "https://github.com/nasa/fprime"
```

Comment:
Could I encourage you to include a newline at the end of the file (that would get rid of the strange ⛔ thing and mean that the next person to touch the file w/ various tools won't be blamed for touching the last line in the file.